bokeh/server/server.py

```diff
@@ -482,54 +490,54 @@ class _ServerOpts(Options):
     Note that due to limitations inherent in Tornado, Windows does not support
     ``num_procs`` values greater than one! In this case consider running
     multiple Bokeh server instances behind a load balancer.
-    """)
+    """)  # type: ignore
 
-    address = Nullable(String, help="""
+    address : str | None = Nullable(String, help="""
     The address the server should listen on for HTTP requests.
-    """)
+    """)  # type: ignore
```

Comment:
This is a trial balloon for one possible way to type our properties. If we can develop a better alternative that is not monstrous to implement, then that would be great. But all the alternatives I've seen seem like they would be incredibly invasive (unlike this, which would merely be incredibly tedious, once)

Reply:
The alternatives may not be pleasant, but at least they would give a working solution, whereas this solves nothing. btw. I considered doing this. If you want to keep this on a provisional basis, then at least use specific `type: ignore[error-code]`.

Reply:
Yes this is just for here. But AFAIK it's at least slighty better than nothing in that the type annotation on the class attribute *does* attach (but is not checked against the actual property type it it initialized with). i.e. with the above changes and this following code:
```
s = _ServerOpts({})
s.ssl_password = 10
```
generates the expected mypy error:
```
bokeh/server/server.py:547:18: error: Incompatible types in assignment (expression has type "int", variable has type "Optional[str]")  [assignment]
```
Which is what is desired. I should also be explicit about my priorties in this regard, which is that user experience in modern editors with intellisense is good.